gensim/models/nmf.py

```diff
@@ -624,10 +631,16 @@ def update(self, corpus, chunksize=None, passes=None, eval_every=None):
 
                     chunk_len = len(chunk)
 
-                logger.info(
-                    "PROGRESS: pass %i, at document #%i/%s",
-                    pass_, chunk_idx * chunksize + chunk_len, lencorpus
-                )
+                if np.isinf(lencorpus):
```

Comment:
... and this.

Reply:
@mpenkov what was up with this line, do you remember?

Reply:
Not 100% certain, but I think my problem with these changes were they add very little value. The benefit does not justify the cost.

Benefit: the log message is prettier, it doesn't contain the corpus length when it's infinity.
Cost: the corresponding logging code is more than twice as long (20 lines instead of just 6).